merkle-tree/indexed/src/copy.rs

```diff
@@ -56,13 +55,29 @@ where
         let indexed_changelog_metadata: CyclicBoundedVecMetadata =
             unsafe { read_value_at(bytes, &mut offset) };
 
+        println!("merkle_tree.height: {:?}", merkle_tree.height);
```

Comment:
1. Leftover `println!`, pls remove - this one and the others in this file.
2. I guess you added these to make sure that the copy wrapper works correctly. Did you spot anything sketchy?

Reply:
>  Did you spot anything sketchy?

No, lgtm!